.github/workflows/docker-image.yml

```diff
@@ -1,23 +1,38 @@
 name: Docker Image
 
 on:
-  push:
-    branches:
-      - master
+  release:
+    types: [created]
```

Comment:
We will need to have both on release and on push here, as users may want to use the nightly version as well

Reply:
the nightly version should be only for python 3.8, or for all python versions?

Reply:
Why not all?